### ISSUE TITLE
wails.localhost refuses to connect on application startup for about a second.

### ISSUE THREAD

**Issue by muirch** [wailsapp/wails#2521]
### Description

After building and starting the application, a message appears for about a second saying that wails.localhost refuses to connect.

### To Reproduce

1. Create new app with vue-ts template:
`wails init -n myproject -t vue-ts`
2. Go to it:
`cd myproject`
3. Run IDE
`goland .`
4. Build the app:
`wails build`
5. Run it.

### Expected behaviour

Instant display of the home page.

### Screenshots

![image](https://user-images.githubusercontent.com/20345694/226902519-3b3d7c43-c8ea-4146-8460-2497b6cb6818.png)

### System Details

```shell
PS C:\Users\mimui\GolandProjects\test> wails doctor
DEB | Using go webview2loader
Wails CLI v2.4.0

Scanning system - Please wait (this may take a long time)...Done.

# System

OS           | Windows 10 Home Single Language
Version      | 2009 (Build: 22621)
ID           | 22H2
Go Version   | go1.20
Platform     | windows
Architecture | amd64

# Wails

Version | v2.4.0

# Dependencies

Dependency | Package Name | Status    | Version
WebView2   | N/A          | Installed | 111.0.1661.44
npm        | N/A          | Installed | 8.19.2
*upx       | N/A          | Available |
*nsis      | N/A          | Available |
* - Optional Dependency

# Diagnosis

Your system is ready for Wails development!
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/
  - nsis : More info at https://wails.io/docs/guides/windows-installer/

 ♥   If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony
```

**Comment by stffabi** [comment]
Thanks for using Wails and reporting this issue.

This issue has been fixed with the `v2.4.1`.

**Comment by stffabi** [comment]
Duplicate of #2506
